From: KevinDuan <duankaiwen@kylinos.cn>
Date: Tue, 23 Jul 2024 11:59:57 +0000
Subject: Translated using Weblate (Kyrgyz)

Currently translated at 79.1% (288 of 364 strings)

Translation: openkylin-nile-new/youker-assistant
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/youker-assistant/ky/
---
 src/translation/kylin-assistant_kk.ts | 2 +-
 src/translation/kylin-assistant_ky.ts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/translation/kylin-assistant_kk.ts b/src/translation/kylin-assistant_kk.ts
index 76b7eaf..cb90685 100644
--- a/src/translation/kylin-assistant_kk.ts
+++ b/src/translation/kylin-assistant_kk.ts
@@ -1224,7 +1224,7 @@
     <message>
         <location filename="../krightwidget.cpp" line="66"/>
         <source>minimize</source>
-        <translation>مىنىمۇملاشتۇرۇش</translation>
+        <translation>ەڭ كشكەنە بەينە</translation>
     </message>
     <message>
         <location filename="../krightwidget.cpp" line="72"/>
diff --git a/src/translation/kylin-assistant_ky.ts b/src/translation/kylin-assistant_ky.ts
index 970c333..009c09a 100644
--- a/src/translation/kylin-assistant_ky.ts
+++ b/src/translation/kylin-assistant_ky.ts
@@ -1224,7 +1224,7 @@
     <message>
         <location filename="../krightwidget.cpp" line="66"/>
         <source>minimize</source>
-        <translation>مىنىمۇملاشتۇرۇش</translation>
+        <translation>ەڭ كىچىك   ابالى</translation>
     </message>
     <message>
         <location filename="../krightwidget.cpp" line="72"/>
